From: "Barak A. Pearlmutter" <barak+git@cs.nuim.ie>
Date: Thu, 1 Sep 2011 20:25:54 +0100
Subject: nofile for hurd

---
 src/Dispatch/dispatcher.cc | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/src/Dispatch/dispatcher.cc b/src/Dispatch/dispatcher.cc
index ab92d0e..b853e19 100644
--- a/src/Dispatch/dispatcher.cc
+++ b/src/Dispatch/dispatcher.cc
@@ -393,6 +393,10 @@ void ChildQueue::notify() {
 #define NOFILE 256
 #endif
 
+#ifndef NOFILE
+#define NOFILE 256
+#endif
+
 Dispatcher::Dispatcher() {
     _nfds = 0;
     _rmask = new FdMask;
