From: "Barak A. Pearlmutter" <barak+git@cs.nuim.ie>
Date: Fri, 30 Jan 2015 01:24:47 +0200
Subject: ofptr scope

---
 src/comterp_/main.cc | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/comterp_/main.cc b/src/comterp_/main.cc
index 3fde6f1..b7a4a64 100644
--- a/src/comterp_/main.cc
+++ b/src/comterp_/main.cc
@@ -137,9 +137,10 @@ int main(int argc, char *argv[]) {
 
     FILE* inptr = argc>=5 ? fopen(argv[4], "r") : stdin;
 
+    FILE* ofptr = nil;
+
     if (!telcat_flag) {
       
-      FILE* ofptr = nil;
       FILEBUF(obuf, ofptr = fdopen(server.get_handle(), "w"), ios_base::out);
       ostream out(&obuf);
       
