Index: ivtools-1.2/config/params.def
===================================================================
--- ivtools-1.2.orig/config/params.def	2009-09-04 17:18:35.000000000 -0400
+++ ivtools-1.2/config/params.def	2009-09-04 17:19:41.000000000 -0400
@@ -285,6 +285,10 @@
 #define CCDriver CC +p			/* might want an absolute path, etc. */
 #endif
 
+#ifndef CCLinker
+#define CCLinker CCDriver
+#endif
+
 #ifndef CCSuffix
 #define CCSuffix c			/* might want cc, cxx, C, etc. */
 #endif
@@ -322,6 +326,7 @@
 #endif
 
            CCDRIVER = CCDriver
+	   CCLINKER = CCLinker
            CCSUFFIX = CCSuffix
             CDRIVER = CDriver
             CSUFFIX = CSuffix
