Index: ivtools-1.2/src/Unidraw/geomobjs.cc
===================================================================
--- ivtools-1.2.orig/src/Unidraw/geomobjs.cc
+++ ivtools-1.2/src/Unidraw/geomobjs.cc
@@ -25,6 +25,8 @@
  * Implementation of geometrical objects used by Graphic.
  */
 
+#include <cstdio>
+
 #include <Unidraw/Graphic/geomobjs.h>
 #include <Unidraw/Graphic/util.h>
 #include <Unidraw/ulist.h>
