Index: ivtools-1.2/src/InterViews/regexp.c
===================================================================
--- ivtools-1.2.orig/src/InterViews/regexp.c	2009-09-05 19:58:05.000000000 -0400
+++ ivtools-1.2/src/InterViews/regexp.c	2009-09-05 19:58:41.000000000 -0400
@@ -26,9 +26,12 @@
  * Regexp - regular expression searching
  */
 
+#include <cstdio>
 #include <InterViews/regexp.h>
 #include <stream.h>
 #include <string.h>
+#include <iostream>
+#include <fstream>
 using std::cerr;
 
 /*
