--- a/src/Time/Date.cc
+++ b/src/Time/Date.cc
@@ -60,6 +60,8 @@
   Modified by Vectaport, Inc.
 */
 
+#include <cstdio>
+
 #include <Time/Date.h>
 #include <Time/Time_.h>
 #include <OS/string.h>
@@ -68,7 +70,10 @@
 using namespace std;
 #include <ctype.h>
 #include <string.h>
-#include <stdio.h>
+#include <iostream>
+
+using std::setfill;
+using std::setw;
 
 #define	THIS	Date
 #define	BASE	Object
