Index: ivtools-1.2/src/IVGlyph/bdvalue.c
===================================================================
--- ivtools-1.2.orig/src/IVGlyph/bdvalue.c	2009-09-04 13:27:22.000000000 -0400
+++ ivtools-1.2/src/IVGlyph/bdvalue.c	2009-09-04 13:27:38.000000000 -0400
@@ -46,8 +46,8 @@
  */
 /* derived from idemo/main.c */
 
-#include <IVGlyph/bdvalue.h>
 #include <cstdio>
+#include <IVGlyph/bdvalue.h>
 #include <string.h>
 
 /* class BoundedValue */
