Index: ivtools-1.2/src/InterViews/field.cc
===================================================================
--- ivtools-1.2.orig/src/InterViews/field.cc
+++ ivtools-1.2/src/InterViews/field.cc
@@ -26,6 +26,8 @@
  * FieldEditor -- simple editor for text fields
  */
 
+#include <cstdio>
+
 #include <Dispatch/dispatcher.h>
 #include <Dispatch/iocallback.h>
 #include <IV-look/field.h>
@@ -60,7 +62,6 @@
 
 #include <OS/math.h>
 #include <OS/string.h>
-#include <cstdio>
 #include <string.h>
 
 class FieldStringEditor : public StringEditor {
