Index: ivtools-1.2/src/InterViews/kit.c
===================================================================
--- ivtools-1.2.orig/src/InterViews/kit.c	2009-09-04 13:45:12.000000000 -0400
+++ ivtools-1.2/src/InterViews/kit.c	2009-09-04 13:45:35.000000000 -0400
@@ -26,6 +26,8 @@
  * WidgetKit -- object for creating common UI objects
  */
 
+#include <cstdio>
+
 #ifdef motif_kit
 #include <IV-look/mf_kit.h>
 #endif
@@ -71,7 +73,6 @@
 #include <OS/list.h>
 #include <OS/string.h>
 #include <OS/ustring.h>
-#include <cstdio>
 
 declareActionCallback(Session)
 implementActionCallback(Session)
