Index: ivtools-1.2/src/InterViews/ol_kit.cc
===================================================================
--- ivtools-1.2.orig/src/InterViews/ol_kit.cc
+++ ivtools-1.2/src/InterViews/ol_kit.cc
@@ -22,6 +22,7 @@
  * OF THIS SOFTWARE.
  */
 
+#include <cstdio>
 #include <Dispatch/dispatcher.h>
 #include <Dispatch/iocallback.h>
 #include <IV-look/bevel.h>
@@ -43,7 +44,6 @@
 #include <InterViews/telltale.h>
 #include <InterViews/window.h>
 #include <OS/string.h>
-#include <cstdio>
 
 /*
  * character definitions
