Index: ivtools-1.2/src/include/ivstd/fstream.h
===================================================================
--- ivtools-1.2.orig/src/include/ivstd/fstream.h	2009-09-04 14:22:19.000000000 -0400
+++ ivtools-1.2/src/include/ivstd/fstream.h	2009-09-04 14:22:32.000000000 -0400
@@ -1,5 +1,6 @@
 #ifndef _iv_fstream_
 #define _iv_fstream_
+#include <cstdio>
 #include_next <fstream>
 #if __GNUC__>=3
 #define input ios_base::in
