Index: ivtools-1.2/src/ComTerp/comterp.c
===================================================================
--- ivtools-1.2.orig/src/ComTerp/comterp.c	2009-09-05 08:51:26.000000000 -0400
+++ ivtools-1.2/src/ComTerp/comterp.c	2009-09-05 08:52:00.000000000 -0400
@@ -196,7 +196,9 @@
     save_parser_client();    
     postfix_echo();
 
-    return status==0 && _pfbuf[_pfnum-1].type != TOK_EOF && _buffer[0] != '\0';
+    return status==0
+      && (_pfnum==0 || _pfbuf[_pfnum-1].type != TOK_EOF)
+      && _buffer[0] != '\0';
 }
 
 boolean ComTerp::eof() {
