Pointer length is different in 32 and 64 bit arches.

--- a/src/Attribute/attrvalue.c
+++ b/src/Attribute/attrvalue.c
@@ -393,7 +393,7 @@ boolean AttributeValue::boolean_val() {
     case AttributeValue::StringType:
 	return (boolean) int_val()!=-1;
     case AttributeValue::ObjectType:
-	return (boolean) obj_val();
+	return (intptr_t) obj_val();
     case AttributeValue::StreamType:
 	return stream_mode() != 0;
     default:
@@ -552,7 +552,7 @@ unsigned int AttributeValue::uint_val()
     case AttributeValue::SymbolType:
 	return (unsigned int) int_val();
     case AttributeValue::ObjectType:
-        return (unsigned int)obj_val();
+        return (unsigned intptr_t) obj_val();
     default:
 	return 0;
     }
@@ -585,7 +585,7 @@ int AttributeValue::int_val() {
     case AttributeValue::SymbolType:
 	return int_ref();
     case AttributeValue::ObjectType:
-        return (int)obj_val();
+      return (intptr_t) obj_val();
     default:
 	return 0;
     }
